archive.php

```diff
@@ -1,40 +1,43 @@
 <?php
+
 /**
  * The template for displaying Archive pages.
  *
  * Used to display archive-type pages if nothing more specific matches a query.
  * For example, puts together date-based pages if no date.php file exists.
  *
- * Learn more: http://codex.wordpress.org/Template_Hierarchy
- *
- * Methods for TimberHelper can be found in the /lib sub-directory
+ * Learn more: https://developer.wordpress.org/themes/basics/template-hierarchy/
  *
- * @package  WordPress
- * @subpackage  Timber
- * @since   Timber 0.2
  */
 
-$templates = array( 'archive.twig', 'index.twig' );
+namespace Timber\StarterTheme;
```

Comment:
If we use `Timber\StarterTheme` as the default namespace, developers might not change it. And using `Timber` as your namespace is probably not correct.

Couldn’t we use `App` or `Theme` as the namespace? And then we should also add it to composer.json [using the `autoload` option](https://getcomposer.org/doc/04-schema.md#autoload).

I already discussed this a bit with Viktor in https://github.com/timber/starter-theme/pull/143/files/d4a882354434270eeb59c56f748edf8ac6c111c4#r1278886080, but didn’t find the right solution yet.

Reply:
Updated all namespaces to `App`. I would like to try to implement the autoloading feature in a separate PR.